twingraph/src/main/kotlin/com/cosmotech/twingraph/api/TwingraphServiceImpl.kt

```diff
@@ -334,9 +292,8 @@ class TwingraphServiceImpl(
       actionLambda: (Map<String, String>) -> Unit
   ): Map<String, String> {
     var map = mapOf<String, String>()
-    inputStream.bufferedReader(UTF_8).use { reader ->
-      val csvFormat: CSVFormat =
-          CSVFormat.DEFAULT.builder().setHeader().setSkipHeaderRecord(false).setTrim(true).build()
+    inputStream.bufferedReader().use { reader ->
```

Comment:
For information purpose, can you tell us why:
- UTF-8 is no longer used
- setSkipHeaderRecord is no longer used
- setTrim is no longer used

Reply:
I will keep only trim. Others options are by default